gc.c

```diff
@@ -11460,6 +11461,10 @@ gc_stress_get(rb_execution_context_t *ec, VALUE self)
 static void
 gc_stress_set(rb_objspace_t *objspace, VALUE flag)
 {
+    if (flag != Qtrue && flag != Qfalse) {
+        flag = rb_to_int(flag);
```

Comment:
GC won't matter by converting to a `Fixnum`.

```suggestion
        flag = UINT2NUM(NUM2UINT(flag));
```